Editor/TooltipsEditor.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
```

Comment:
I haven't tried this out yet directly, but am wondering what the custom editor delivers?